go-controller/hack/verify-gofmt.sh

```diff
@@ -6,19 +6,36 @@ set -o pipefail
 
 PKGS=${PKGS:-.}
 
-find_files() {
-  find ${PKGS} -not \( \
+find_files="find ${PKGS} -not \( \
       \( \
         -wholename '*/vendor/*' \
         -o -wholename '*/_output/*' \
       \) -prune \
-    \) -name '*.go'
-}
+    \) -name '*.go'"
 
 GOFMT="gofmt -s"
-bad_files=$(find_files | xargs $GOFMT -l)
+bad_files=$(eval $find_files | xargs $GOFMT -l)
 if [[ -n "${bad_files}" ]]; then
   echo "!!! '$GOFMT' needs to be run on the following files: "
   echo "${bad_files}"
   exit 1
 fi
+
+rectify_errors=$(echo $find_files \| xargs sed -i -e \''s/\(fmt\.Errorf.*"\)\([A-Z][a-z]\+\s\)/\1\l\2/g'\')
+rectify_logs=$(echo $find_files \| xargs sed -i -e \''s/\(klog\..*"\)\([a-z]\+\s\)/\1\u\2/g'\')
+
+malformatted_errors=$(eval $find_files | xargs sed -n -e 's/\(fmt\.Errorf.*"\)\([A-Z][a-z]\+\s\)/\1\l\2/p')
+malformatted_logs=$(eval $find_files | xargs sed -n -e 's/\(klog\..*"\)\([a-z]\+\s\)/\1\u\2/p')
```

Comment:
(Your second commit is fixing the rules you created in the first commit.)

Reply:
Yes, do you want me to squash them and have only one commit which does all of this? I can do that once the PR is reviewed

Reply:
It's fine to split out the klog fixes and the fmt.Errorf fixes into their own commits, but you shouldn't have fixes to `verify-gofmt.sh` mixed in with the klog fixes; that should all be in the first commit

Reply:
Crap, you're right. This change must have been slipped me by during when re-working my commits/code.